### ISSUE TITLE
Backward compatibility

### ISSUE THREAD

**Issue by Lynn42** [surveyjs/surveyjs#782]
### Are you requesting a feature, reporting a bug or ask a **question**?

Question

We were wondering whether we are right in assuming that the SurveyJS Editor library is backward compatible with older versions of the Survey.JS. Libary.
So for instance, if we would use a new version of the Editor to build a form,  and then try to show it using an older version of the Survey.JS library (for existing projects), would that create a problem?

**Comment by andrewtelnov** [comment]
@Lynn42 
SurveyJS Editor should able to edit all survey jsons, that has the same or early versions.
SurveyJS Library should be able to run  all survey jsons, that has the same or early versions.
We fully support the very first version of survey JSON that has been released almost two years ago by our last version, v0.96.1 at the current moment.
We want to continue doing it.

However if survey json has been created by the latest version of Survey Editor and you do not have the latest version of SurveyJS library in run-time, the new features will be simply ignored.
For example. We have added QuestionExpression 2-3 weeks ago. If you add this question type into your survey and try to run it by SurveyJS library, that has been released 2 months ago, the library will just ignore type "expression", since there is no QuestionExpression in it. It doesn't know what object should it create. 

Thank you,
Andrew
